util/src/main/scala/org/ensime/vfs/EnsimeVFS.scala

```diff
@@ -46,6 +46,22 @@ object `package` {
   private val zipFileField = classOf[ZipFileSystem].getDeclaredField("zipFile")
   zipFileField.setAccessible(true)
 
+  object URLParamEncoder {
+    def encode(input: String): String = {
```

Comment:
Won't `input.map{ c =>  if ((c == ' ') || (c == '%')) "%%%02x".format(c.toShort) else c }` do the same thing?